### PR TITLE
NHibernate 5.2.7 and SQL Anywhere 17 support

### PR DESCRIPTION
**Pull request by pettersoft**
NHibernate 5.2.* has support for SQL Anywher 17. Therefore I upgraded to 5.2.7 and added support for SQL Anywhere. All tests are passing.

Not sure how you manage the $(NHibernatePackageVersion) variable. I manually changed it to 5.2.7 using Nuget.

Changed global.json version to 2.1.700, couldn't find the correct version on https://dotnet.microsoft.com/download/visual-studio-sdks

### CODE REVIEW

**Comment by pettersoft** [comment]
Anyone still minding this project?

**Comment by ToGoOrNotToGo** [comment]
It would be great if higher NHibernate releases were supported.

**Comment by igitur** [comment]
I'm not an official reviewer of this project, but I can see a few problems with this PR. 

Firstly, you're mixing 2 different changes in 1 commit. Ideally they should be separate PRs, but at the least separate commits.

Secondly, it's pretty clear that the NHibernate verison is coming from a different configuration source, specifically in `src/Shared.msbuild`, so replacing them in each project file with a hardcoded version is not going to pass a review.

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "2.1.4"
+        "version": "2.1.700"
```
Comment (igitur) [inline global.json:4]: Is this change intended?

File: src/Examples.FirstAutomappedProject/Examples.FirstAutomappedProject.csproj [hunk 2]
```diff
@@ -15,7 +15,7 @@
   
   <ItemGroup>    
     <PackageReference Include="EntityFramework" Version="6.2.0" />    
-    <PackageReference Include="NHibernate" Version="$(NHibernatePackageVersion)" />    
+    <PackageReference Include="NHibernate" Version="5.2.7" />
```
Comment (igitur) [inline src/Examples.FirstAutomappedProject/Examples.FirstAutomappedProject.csproj:18]: Revert all these lines.
